include/certifier_framework.h

```diff
@@ -234,11 +234,20 @@ class cc_trust_manager {
 
   // Each of the enclave types have bespoke initialization
 
+  // Interface ignored through SWIG bindings
```

Comment:
The `%ignore` directive comes in `certifier_framework.h` .

Reply:
Note: Cross-check if with the ignore directive do we still need this shell script.